docs/src/main/asciidoc/hibernate-search-guide.adoc

```diff
@@ -475,8 +475,8 @@ In our existing `LibraryResource`, we just need to inject the following methods
     @GET
     @Path("author/search")
     @Transactional
-    public List<Author> searchAuthors(@QueryParam("pattern") String pattern, // <3>
-            @QueryParam("size") Optional<Integer> size) {
+    public List<Author> searchAuthors(@QueryParam String pattern, // <3>
```

Comment:
Could you add in the callout that the `@QueryParam` here is the one from `org.jboss.resteasy.annotations.jaxrs.`?

Reply:
Done.